docs/theme_architecture/blocks/schema/variables/variable_types/experience_date.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Experience Date
+parent: Variable types
+has_children: false
+---
+
+# Experience Date
+
+Renders two select inputs from which a Creator can select an experience and then a date that has availability for the chosen experience.
+
+Returns an [Experience Slot Calendar Date]({% link docs/reference/objects/experience_slot_calendar/date.md %}) object.
+
+- Accepts a `default` of `next_upcoming` which will select the next available date from the Creator's published experiences.
```

Comment:
**nitpick** I don't think this sentence needs a bullet point

![Screenshot 2023-11-27 at 09 16 09](https://github.com/easolhq/easolhq.github.io/assets/10531930/63fba330-6f63-41fe-96b3-db306521bd8c)